problems/removeEvenStrings.js

```diff
@@ -19,8 +19,15 @@
 * returns: []
 */
 
-function removeEvenStrings() {
-
+function removeEvenStrings(arr) {
+    let newArr = []
+    for(let i = 0; i < arr.length; n++){
+        if(i[0] % 2 === 1){
```

Comment:
Recall how to index into arrays. What is the variable name for our array and what is the variable name for our index?